sympy/geometry/ellipse.py

```diff
@@ -323,6 +322,39 @@ def circumference(self):
         else:
             return 4 * self.major * elliptic_e(self.eccentricity**2)
 
+    def ellipse_circumference(self):
+        """
+        Computes the circumference of an ellipse using length of semi-axes as a and b.
+        Require a >= 0 and b >= 0. This is much faster than the circumference method
+        given in the method circumference.
+        Relative accuracy is about 0.5 ** 53 and computes result numerically using
+        double precision.
+
+        Examples
+        ========
+        >>> from sympy import Ellipse, Point
+        >>> e = Ellipse(Point(0,0), 4, 3)
+        >>> e.ellipse_circumference()
+        22.1034921607095
+        >>> e.circumference.n()
+        22.1034921607095
+        """
+
+        import math
```

Comment:
You should rather import sqrt, pi, etc from sympy, no need to import math.

Reply:
@Abdullahjavednesar   will do that.